build.gradle

```diff
@@ -42,7 +42,7 @@ task coverage(type: JacocoReport) {
 
 dependencies {
     String jUnitVersion = '5.4.0'
-    String javaFxVersion = '11'
+    String javaFxVersion = '16'
```

Comment:
Why is the javaFXVersion changed to 16?

Reply:
I changed the javaFxVersion cos the old version didn't work on my Mac. This is the same method which was recommended in the forum to deal with gibberish appearing on the UI!

Reply:
You might want to consider downloading the OpenJDK Java 11 instead of the Oracle Java 11. Faced the same issue previously as well. You can take a look here: https://nus-cs2103-ay2122s2.github.io/website/admin/programmingLanguages.html

Reply:
I have changed the javaFxVersion back! Thank you!